libs/orchestrator-ui-components/src/lib/components/Table/Table.tsx

```diff
@@ -20,7 +20,7 @@ import {
 } from '@elastic/eui/src/components/datagrid/data_grid_types';
 
 // Total height of grid button bar, table header and pagination bar
-const EUI_DATA_GRID_HEIGHT_OFFSET = 103;
+const EUI_DATA_GRID_HEIGHT_OFFSET = 124;
 
 const GRID_STYLE: EuiDataGridStyle = {
```

Comment:
Out of scope: `Table` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.